index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="style.css">
+        <title>finish-mobile-version</title>
+    </head>
+<body>
+    <div class="grid-container">
```

Comment:
- [ ] Please be more clear, and polite in your comments and try to tell why students should follow your advice. It seems like you are ordering the student for something and even your comment is not clear to understand. Kindly use some polite words with some friendly emoji to write a review comment.

### Your Comment

> Good job using GRID CONTAINER. However, you need to finish your code. in unclear at this time.

Reply:
Im really impressed with your advance!! Well done! 🚗 . However, you can finish your code As suggested in the exercise.

you can get more information in:

https://www.w3schools.com/css/css_grid_container.asp

👍 👍 👍 👍 👍 👍 👍 👍